packages/postcss-minify-selectors/src/index.js

```diff
@@ -163,7 +163,7 @@ const tagReplacements = new Map([
 function tag(selector) {
   const value = selector.value.toLowerCase();
 
-  if (tagReplacements.has(value)) {
+  if (tagReplacements.has(value) && selector.parent && selector.parent.nodes.length === 1) {
```

Comment:
Apologies for the long delay in answering. It looks fine, but could you maybe write a comment explaining what the `selector.parent` etc. check is trying to achieve?

Reply:
Sure. I refactored and added comments to clarify the handling of simple vs complex tag selectors. https://github.com/cssnano/cssnano/pull/1518/commits/e7596d65d03a74aafde86424322fec84712c10ca

I'm not sure if checking parent node length is the best way to differentiate between a complex or a simple selector. Do you know if there is a better way?